astroid/nodes/node_ng.py

```diff
@@ -206,6 +206,13 @@ def last_child(self) -> Optional["NodeNG"]:
             return attr
         return None
 
+    def node_ancestors(self):
```

Comment:
Unfortunately `ClassDef` already has an `ancestors` method.

Reply:
Renaming that one to `class_ancestors` and making this one `ancestors` would be great, but it would take some work.

Reply:
This would require deprecation, do you think it's worth it @cdce8c ?

Reply:
I should add that I don't feel like doing that work, so it's not something I'm pushing for!

Reply:
Initially I also though that `ancestors` would be a good name. Didn't though about the name conflict though.

As for changing the existing one to `class_ancestors`: Not sure it will make much of a difference. Only one method can be named `ancestors` after all. `node_ancestors` is still a pretty good name.